index.html

```diff
@@ -53,7 +53,255 @@ <h3 class="link-item">
           </div>
         </div>
       </section>
-      
-    </main>
+     <section id="myworks">
```

Comment:
- [ ] Please align the ul list items to the center 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_8](https://user-images.githubusercontent.com/69638013/158976338-07794860-d13d-41c8-86ca-fe57b6a2fef5.png) | ![CENTER](https://user-images.githubusercontent.com/69638013/158976294-a7aede84-e1f2-4002-8689-3c9ea236848c.png)

- [ ] It would be nice to add the background border to each project card 👍🏿